components/collector/src/collector_utilities/date_time.py

```diff
@@ -0,0 +1,38 @@
+"""Date and time utilities."""
+
+from datetime import datetime
+from typing import Final
+
+from dateutil.parser import parse
+from dateutil.tz import tzlocal
+
+
+MIN_DATETIME: Final = datetime.min.replace(tzinfo=tzlocal())
+MAX_DATETIME: Final = datetime.max.replace(tzinfo=tzlocal())
+
+
+def days_ago(date_time: datetime) -> int:
```

Comment:
I don't think I've seen a test file that explicitly tests these utilities. Perhaps I looked over it, but we should really have one!

Reply:
Fixed.